File: Src/Client/src/app/shared/models/activity-group.model.ts
import { ActivityGroupUserModel } from './activity-group-user.model';
import { ActivityModel } from './activity.model';
export class ActivityGroupModel {
  id: string;
  projectId: number;
  name: string;
  description: string;
  activities: ActivityModel[];
  activityGroupUserModels: ActivityGroupUserModel[];
}

File: Src/Server/Kloon.EmployeePerformance.Models/TimeSheet/ActivityGroupModel.cs
using System;
using System.Collections.Generic;

namespace Kloon.EmployeePerformance.Models.TimeSheet
{
    public class ActivityGroupModel
    {
        public Guid Id { get; set; }
        public int? ProjectId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public List<ActivityModel> Activities { get; set; }
        public List<ActivityGroupUserModel> ActivityGroupUserModels { get; set; }
    }
}
